### DISCUSSION TITLE
Is it possible to limit the times available in V-Text-Field with type="time"?

### DISCUSSION THREAD

**Discussion by WH-Dev-Acc** [vuetifyjs/vuetify#18185] [Q&A] [answered]
Is it possible to limit the times available in V-Text-Field with type="time"?
I have two of these components, starttime and endtime.
I want to limit the available times in the 2nd, to only show times after the starttime.
Is this possible?
Thank you

**Comment by KaelWD** [comment, marked as answer]
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time#additional_attributes

**Reply by WH-Dev-Acc** [reply]
I think the list option may work for me, I didnt know about that one
I had thought min/max would do it, but not for what I want. 
Thank you ill give it a try.
Hopefully works dynamically

**Reply by WH-Dev-Acc** [reply]
Okay, so I have been doing some checking and I cannot find a way to filter out the times I do not want using the list attribute, only add more.
I could add all the acceptable times, but there is still an 'other' option that appears, which contains all the hours and minutes.
I am leaving this as the accepted answer however as it did answer my question for me. 
Thank you
